contracts/modules/capital/SwapOperator.sol

```diff
@@ -22,10 +22,19 @@ contract SwapOperator is ReentrancyGuard {
     uint maxSlippageRatio;
   }
 
-  ITwapOracle public twapOracle;
-  address public swapController;
-  INXMMaster master;
-  address public stETH;
+  /* storage */
+  bool public communityFundTransferExecuted;
+
+  /* immutables */
```

Comment:
The changes to the variables were:
1. Reorder so we'd have all variables/constants near each other
2. Make variables initialized in constructor immutable
3. Add `communityFundTransferExecuted` storage variable